frontend/src/metabase/admin/datamodel/utils/humanizeCoercionStrategy.ts

```diff
@@ -0,0 +1,43 @@
+const LEFT_TERM_CONVERSIONS: Record<string, string> = {
+  ISO8601: "ISO 8601",
+  UNIXSeconds: "UNIX seconds",
+  UNIXMilliSeconds: "UNIX milliseconds",
+  UNIXMicroSeconds: "UNIX microseconds",
+  UNIXNanoSeconds: "UNIX nanoseconds",
+  YYYYMMDDHHMMSSString: "YYYYMMDDHHMMSS string",
+  YYYYMMDDHHMMSSBytes: "YYYYMMDDHHMMSS bytes",
+};
+
+const RIGHT_TERM_CONVERSIONS: Record<string, string> = {
+  DateTime: "Datetime",
+};
```

Comment:
Could we localize these strings?

ttag won't know the locale if these Records are defined in module scope, so you could define them inside a `getStrings` function called by the treat* functions